.circleci/config.yml

```diff
@@ -201,33 +201,37 @@ workflows:
           <<: *only_branches
       - test:
           <<: *only_branches
-      - security-code:
-          name: Snyk code
-          context:
-            - snyk
-          <<: *only_branches
-      - security-oss:
-          name: Snyk test
-          context:
-            - snyk
-          <<: *only_branches
-  pullrequest_acc:
-    jobs:
-      - hold:
-          type: approval # presents manual approval button in the UI
-      - test_acc:
-          name: "Acceptance tests: << matrix.pattern >>"
-          requires:
-            - hold
-          matrix:
-            parameters:
-              pattern:
-                - TestAcc_Aws
-                - TestAcc_Github_
-                - TestAcc_Google
-                - TestAcc_Azure_
-          context:
-            - driftctl-acc
+# Disabled since it fail for contributors because of the snyk token stored in a private context
```

Comment:
Instead of disabling them, isn't there a way to skip them when the context cannot be shared ?

Reply:
I was looking for something like that initially, but I didn't manage to find something. Feel free if you want to have a look, no emergency for that

Reply:
Yeah too bad CircleCI didn't address this issue yet